### PR TITLE
Init CI for Pull Requests

### PR DESCRIPTION
**Pull request by NoranPeLage**

### CODE REVIEW

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@bycedric/use-expo?release-channel=pr-1).

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - uses: expo/expo-github-action@v5
+        with:
+          expo-version: 3.x
+          expo-username: ${{ secrets.EXPO_CLI_USERNAME }}
+          expo-password: ${{ secrets.EXPO_CLI_PASSWORD }}
+      - run: yarn install
+      - run: expo publish --release-channel=pr-${{ github.event.number }}
```
Comment (ThomasCarca) [inline .github/workflows/pr.yml:18]: Maybe having a branch name in the release channel name would be a good idea ?

Comment (ThomasCarca) [inline .github/workflows/pr.yml:18]: Maybe having a branch name in the release channel name would be a good idea ?

**Review by ThomasCarca — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (ThomasCarca) [inline .github/workflows/pr.yml:18]: Maybe having a branch name in the release channel name would be a good idea ?

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - uses: expo/expo-github-action@v5
+        with:
+          expo-version: 3.x
+          expo-username: ${{ secrets.EXPO_CLI_USERNAME }}
```
Comment (ThomasCarca) [inline .github/workflows/pr.yml:15]: azd

**Review by ThomasCarca — CHANGES_REQUESTED** [review]

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@noranpelage/match-and-watch?release-channel=pr-1).

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@noranpelage/match-and-watch?release-channel=pr-1).

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@noranpelage/match-and-watch?release-channel=pr-1).

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@noranpelage/match-and-watch?release-channel=pr-${GITHUB_REF##*/}).

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@noranpelage/match-and-watch?release-channel=pr-).

**Review by tfonroques — DISMISSED** [review]

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@noranpelage/match-and-watch?release-channel=pr-1).

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@noranpelage/match-and-watch?release-channel=pr-1).

**Comment by NoranPeLage** [comment]
App is ready for review, you can [see it here](https://expo.io/@noranpelage/match-and-watch?release-channel=pr-1).

**Review by ThomasCarca — DISMISSED** [review]

**Review by ThomasCarca — APPROVED** [review]

**Review by tfonroques — APPROVED** [review]

**Pull request merged**